app.py

```diff
@@ -0,0 +1,138 @@
+import base64
+import json
+import logging
+import os
+import sys
+import boto3
+
+from chalice import Chalice
+
+app_name = 'CMSFilterApp'
+
+app = Chalice(app_name=app_name)
+
+# Initialize the logger
+logging.basicConfig(filename='app.log', filemode='w', format='%(name)s - %(levelname)s - %(message)s')
+logger = logging.getLogger(app_name)
+
+# Set the log level from Environment variable LOG_LEVEL
+if os.environ.get('LOG_LEVEL') is None:
+    logger.setLevel(logging.DEBUG)
+else:
+    logger.setLevel(os.environ['LOG_LEVEL'])
+
+kinesis_endpoint_url = os.environ.get('KINESIS_ENDPOINT_URL')
+aws_access_key_id = os.environ.get('AWS_ACCESS_KEY_ID', 'iam')
+aws_secret_access_key = os.environ.get('AWS_SECRET_ACCESS_KEY', 'iam')
+aws_region = os.environ.get('AWS_REGION', 'us-east-1')
+output_stream_name = os.environ.get('OUTPUT_STREAM')
+logger.debug('##Environment variables {}'.format(os.environ))
+
+if os.environ['OUTPUT_STREAM'] is None:
+    raise KeyError('Output stream is required before running the lambda!!')
+
+
+# function to create a client with aws for a specific service and region
+def create_client(service):
+    return boto3.client(
+        service,
+        endpoint_url=kinesis_endpoint_url,
+        aws_access_key_id=aws_access_key_id,
+        aws_secret_access_key=aws_secret_access_key,
+        region_name=aws_region)
+
+
+# function for sending data to Kinesis at the absolute maximum throughput
+def send_kinesis(kinesis_client, kinesis_stream_name, kinesis_record):
+    kinesisRecords = [kinesis_record]
+    # put the records to kinesis
+    kinesis_client.put_records(
+        Records=kinesisRecords,
+        StreamName=kinesis_stream_name
+    )
+
+
+def parse_kinesis_record(record):
```

Comment:
this isn't parsing, this is decoding, we should probably just call it decode

Reply:
hmm. I'll rename it as "decode_record" as decode is already a predefined function from base64 we are using in this method only.